r124/outref/ydb312_gtm8182e.txt

```diff
@@ -0,0 +1,56 @@
+# Create the DB
+
+
+# Start INST1 INST2 replication
+==Executing MULTISITE_REPLIC 'START INST1 INST2'==
+Starting Primary Source Server in ##TEST_PATH##
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
+
+# Start INST3 INST4 replication
+==Executing MULTISITE_REPLIC 'START INST3 INST4'==
+Starting Primary Source Server in ##FILTERED##_REMOTE_TEST_PATH_/instance3
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance4
+
+
+# Running rmv_map script in INST3 to remove instance file mapping
+==Executing MULTISITE_REPLIC 'RUN INST3 ##TEST_COM_PATH##/rmv_map.csh'==
+%GDE-I-LOADGD, Loading Global Directory file 
+	##REMOTE_TEST_PATH##/instance3/mumps.gld
+%GDE-I-VERIFY, Verification OK
+
+%GDE-I-VERIFY, Verification OK
+
+%GDE-I-GDUPDATE, Updating Global Directory file 
+	##REMOTE_TEST_PATH##/instance3/mumps.gld
+==Executing MULTISITE_REPLIC 'RUN INST3 ##SOURCE_PATH##/mumps -run GDE SHOW -INSTANCE  '==
+%GDE-I-LOADGD, Loading Global Directory file 
+	##REMOTE_TEST_PATH##/instance3/mumps.gld
+%GDE-I-VERIFY, Verification OK
+
+%GDE-I-NOACTION, Not updating Global Directory ##REMOTE_TEST_PATH##/instance3/mumps.gld
+
+
+# Copy ydb312gtm8182e.m to INST3
+# Unset [ydb/gtm]_repl_instance env variables  and run ydb312gtm8182e.m to set globals on INST1 and INST3 from INST3
+==Executing MULTISITE_REPLIC 'RUN INST3 unsetenv ydb_repl_instance; unsetenv gtm_repl_instance; $gtm_dist/mumps -run ydb312gtm8182e'==
+    Change to INST1 global directory and set ^a
+    Change to mumps.gld global directory and set ^a
```

Comment:
I would like this to go to INST3 global directory instead of mumps.gld. Makes it similar to line 36.